client/src/components/DataFiles/DataFilesProjectFileListing/DataFilesProjectFileListing.module.scss

```diff
@@ -23,10 +23,7 @@
   white-space: pre-wrap;
 
   /* FAQ: `14px` is part of normalization of "table" font sizes */
-  font-size: 0.875rem; /* 14px (assumed deviation from design) */
```

Comment:
Any reason for this change from rem to px?

Reply:
Ah I was playing around. But forgot to change it.